packages/hooks/src/useActiveWallet.ts

```diff
@@ -1,10 +1,10 @@
 import { useMemo } from 'react';
-import useWalletsList from './useWalletsList';
+import useWalletAccountsList from './useWalletAccountsList';
 
 /** A custom hook that returns the wallet object for the current active wallet. */
```

Comment:
@sergei-deriv Should we mark this hook as `deprecated` with `useActiveWalletAccount`? 🤔